highlight/events-explainer.md

```diff
@@ -27,7 +27,7 @@ interface HighlightPointerEvent : PointerEvent {
 }
 ```
 
-For the selected range, a HighlightPointerEvent object must be dispatched with the Highlight as its `currentTarget`, and the relevant range as its `range`. The event's `target` will be set to the element the user interacted with.
+For the selected highlight/range pair, a HighlightPointerEvent object must be dispatched with the relevant range as its `range` and the Highlight as its `currentTarget`. `Highlight` will be based on `EventTarget` in order to be able to have events fired against it. The event's `target` will be set to the element the user interacted with.

```

Comment:
Regarding:
> `Highlight` will be based on `EventTarget` in order to be able to have events fired against it.

It might be more precise to state this as:
> The `Highlight` interface will be changed to inherit from [`EventTarget`](https://dom.spec.whatwg.org/#eventtarget) in order to be able to have event listeners added to it and events fired against it.

(The DOM spec uses "inherit" rather than "based on" when discussing these relationships).

I also think this change is important enough to split this sentence into a separate paragraph and include the updated IDL. The IDL from https://drafts.csswg.org/css-highlight-api-1/#custom-highlight will change to:
```
[Exposed=Window]
interface Highlight : EventTarget {
  constructor(AbstractRange... initialRanges);
  setlike<AbstractRange>;
  attribute long priority;
  attribute HighlightType type;
};
```